tensorflow_serving/tools/docker/Dockerfile.gpu

```diff
@@ -60,6 +60,8 @@ RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machi
     && apt-get clean && \
     rm -rf /var/lib/apt/lists/*;
 
+RUN apt-get update && apt-get install -y cuda-toolkit-11-8
```

Comment:
To be conscious about the size of the container image, I would recommend installing this as part of the RUN command at line 34 to prevent additional Docker layers.